echo-of-the-song/Assets/Game/Scripts/Damage/DetectionData.cs

```diff
@@ -0,0 +1,8 @@
+using UnityEngine;
+
+public class DetectionData
```

Comment:
Add the field for detected object
example :
public class DetectionData<T>
{
public T detectedObject;
........
}